src/main/proto/wfa/measurement/protocol/liquid_legions_sketch_aggregation_v2.proto

```diff
@@ -122,11 +106,11 @@ message LiquidLegionsSketchAggregationV2 {
     // the aggregator. It is only set at the aggregator and is not shared with
     // non-aggregator nodes. The value will be included in the result for the
     // computation.
-    ReachEstimate reach_estimate = 5;
```

Comment:
We should start following the guidelines for protocol buffers, i.e. don't change the field numbers
https://developers.google.com/protocol-buffers/docs/proto#updating

Reply:
I think that guidance is for code already running in production. Whenever possible, I feel a little better to make the code cleaner.

Possibly we should enforce that whenever there is a proto type or field change, the PR should explicitly have a description like "TYPE_CHANGE_OK=code_under_development"